roles/ci_gen_kustomize_values/README.md

```diff
@@ -39,9 +39,26 @@ This configMap needs some more parameters in order to properly override the `arc
 * `cifmw_ci_gen_kustomize_values_ssh_public_key`: (String) SSH public key associated to `cifmw_ci_gen_kustomize_values_ssh_private_key`.
 * `cifmw_ci_gen_kustomize_values_migration_priv_key`: (String) SSH private key dedicated for the nova-migration services.
 * `cifmw_ci_gen_kustomize_values_migration_pub_key`: (String) SSH public key associated to `cifmw_ci_gen_kustomize_values_migration_priv_key`.
+* `cifmw_ci_gen_kustomize_values_provisioning_interface`: (String) Used to override default provisioning interface from OCP nodes.
+* `cifmw_ci_gen_kustomize_values_root_password`: (String) Used to override default baremetal compute root password.
+* `cifmw_ci_gen_kustomize_values_os_net_configs_mappings`: (Dictionary) Used to override default os-net-config nic mappings.
+* `cifmw_ci_gen_kustomize_values_network_config_template`: (List) Used to override default os-net-config network config template.
 
 Note that all of the SSH keys should be in `ecdsa` format to comply with FIPS directives.
 
+NFV parameters in order to properly override the `architecture` provided one based on compute node.
```

Comment:
We don't go that deep in this role, if the user wants to override those values, why not pathing the output using `cifmw_architecture_user_kustomize`?

Reply:
@pablintino ack